src/db/collections/analytics/milestone_size.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#[derive(Clone, Debug, Default)]
+#[allow(missing_docs)]
+pub struct MilestoneSizeAnalyticsResult {
+    pub total_milestone_payload_bytes: u64,
+    pub total_tagged_data_payload_bytes: u64,
+    pub total_transaction_payload_bytes: u64,
+    pub total_treasury_transaction_payload_bytes: u64,
+    pub total_milestone_bytes: u64,
```

Comment:
why `u64`? I thought we'ld use `u64` only for values and `usize` for counts?

Reply:
These are removed in another PR.